### DISCUSSION TITLE
Available IP Addresses Not Displayed correctly

### DISCUSSION THREAD

**Discussion by k86td** [netbox-community/netbox#12887] [Q&A] [answered]
Hello! I'm provisionning IP addresses and I'm not sure if the table should look like this or not.

![image](https://github.com/netbox-community/netbox/assets/40375233/9358c402-095c-4570-b962-013f530e86e8)

This is when I click on a prefix that has many child IP addresses inside. The real IPs show up at the last pages.

Is there something I can do?

**Comment by abhi1693** [comment, marked as answer]
You have sorted the column by ip address. If you click on the red cross, the column should revert to normal display.

**Reply by k86td** [reply]
Yeah I figured it out after a while. Thanks anyway:)
